airflow/www/package.json

```diff
@@ -60,6 +60,7 @@
     "bootstrap-3-typeahead": "^4.0.2",
     "codemirror": "^5.59.1",
     "d3": "^3.4.4",
+    "d3-shape": "^2.1.0",
```

Comment:
This one is a BSD license: https://github.com/d3/d3-shape/blob/master/LICENSE

Let me see if we need to add a license somewhere

Reply:
We need to include https://github.com/d3/d3-shape/blob/master/LICENSE license file in 

https://github.com/apache/airflow/tree/master/licenses

Reply:
Done!